lib/diagnostic.rb

```diff
@@ -108,10 +110,10 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'person = {first_name: "Alex", favorite_number: 7}'
 
 # Set a default return value to your `person` hash that concatenates the value
 # of person[:first_name] and a message about them.
 # Write your response as a Ruby string.
 
-Diagnostic.hash_default = 'your answer here'
+Diagnostic.hash_default = 'person = Hash.new { |hash, key| hash[key] = "Go Fish: #{person[:first_name]}" }''
```

Comment:
Here it's
`person.default = "#{person[:first_name]} is awesome"` (of course you could use any sting at the end